Apps/Laboratory/src/Services/LabTestKitService.cs

```diff
@@ -172,28 +172,33 @@ private static void ProcessResponse<T>(RequestResult<T> requestResult, HttpRespo
                 case HttpStatusCode.OK:
                     requestResult.ResultStatus = ResultType.Success;
                     break;
+
```

Comment:
Last month there were several tasks to clean up warnings in the code base.  Having an unnecessary space was one of them.  Lines 180, 185, 193 and 281.

<img width="2051" alt="Screenshot 2022-12-22 at 1 30 05 PM" src="https://user-images.githubusercontent.com/58790456/209229888-43105e3d-c2ec-4228-a01f-b4e87961142b.png">